lib/src/schedules.dart

```diff
@@ -35,6 +35,55 @@ sealed class Schedule {
 
     return true;
   }
+
+  List<DateTime> findNextNOccurrences(Schedule schedule, int n,
```

Comment:
Do we need to pass in a `Schedule`? Can we just use the current instance?

This same question applies to `findNextTilDateOccurrences`, too.

Reply:
Yes is possible to use the current instance, i just added as a static methos to be as less invasive as possible.

Reply:
I hear ya. No rush!

I agree that calculating instead of brute-forcing it would be a nice improvement. Once this is merged, I might play around with the idea.